src/Migrations/Version20250717121454.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of Monsieur Biz' Menu plugin for Sylius.
+ *
+ * (c) Monsieur Biz <sylius@monsieurbiz.com>
+ *
+ * For the full copyright and license information, please view the LICENSE.txt
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace MonsieurBiz\SyliusMenuPlugin\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20250717121454 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('ALTER TABLE monsieurbiz_menu_item ADD enabled TINYINT(1) NOT NULL');
```

Comment:
```suggestion
        $this->addSql('ALTER TABLE monsieurbiz_menu_item ADD enabled TINYINT(1) NOT NULL');
```

Set default value to true to be sure the upgrade of the plugin will be ok
You can add it in the doctrine declaration of the field